newdle/api.py

```diff
@@ -356,6 +358,29 @@ def update_participant(args, code, participant_code):
     for key, value in args.items():
         setattr(participant, key, value)
     db.session.commit()
+    formatted_answers = []
+    for date, state in participant._answers.items():
+        formatted_answers.append(
+            "Date: {} ({})".format(
+                date.replace("T", " at "), state.replace("ifneedbe", "if needed")
+            )
```

Comment:
I don't think this is a good way of doing this.  I would rather use strftime to format the datetime that's in `answers`.

For the state, define a dict mapping state names to human-friendly strings. either here in python or in a Jinja macro used in the templates.